QUEST_LV_0100.tsv

```diff
@@ -6403,101 +6403,101 @@ QUEST_LV_0100_20150317_006402	{memo X}Mr. Juan asked you to bring him Yukopus Le
+QUEST_LV_0100_20150323_006406	$Now take the unsealing stone and go to Ramstis Ridge. {nl}Student of Gustas will teach you how to release the seal there. 
+QUEST_LV_0100_20150323_006407	$Were you not guided yet?{nl}It's the student of Gustas in Ramstis Ridge.{nl}It is made to let only the chosen one enter.
+QUEST_LV_0100_20150323_006408	$Ask the mayor of the Miners' Village for directions to Gele Plateau. {nl}I will sent the message to the Paladin Master first. {nl}May the blessings of Goddess be with you.
 QUEST_LV_0100_20150323_006409	$The high gardens that the Goddess mentioned could be an area within Gele Plateau. {nl}I heard the Paladin Master is there, uploading a long term promise. 
 QUEST_LV_0100_20150323_006410	{memo X}
 QUEST_LV_0100_20150323_006411	{memo X}
 QUEST_LV_0100_20150323_006412	{memo X}If you ever fight against us Rodeleros, better be careful of your legs. {nl}Swords pop out from below the shield. {nl}
-QUEST_LV_0100_20150323_006413	The simplest is the strongest. {nl}If you want to study the roots of everything, take the path of Elementalist.
-QUEST_LV_0100_20150323_006414	Go straight down the Twin Bridge and you'll arrive at Srautas Gorge.{nl}Go further into the right from the Srautas Gorge and you'll be able to get to Gele Plateau.
-QUEST_LV_0100_20150323_006415	The Paladin Master is in Gele Plateau.{nl}You can get there by riding the cable car.
-QUEST_LV_0100_20150323_006416	It is a device that the first Paladin set in preparation for the invasion of the demons. {nl}Many of it are placed to cover the whole of Nefritas Cliff. {nl}
-QUEST_LV_0100_20150323_006417	You need to find Goddess Saule..{nl}The colorful stream mentioned in the revelation must refer to the stream beginning at the Veja Ravine. {nl}Well then, may the blessings of Goddess be with you.
-QUEST_LV_0100_20150323_006418	This is the holy weapon used to fight against Gesti in Nefritas Cliff. {nl}It has been handed down from since the first Paladin.
-QUEST_LV_0100_20150323_006419	Go. Tenet Cathedral is in Tenet Garden.{nl}Be sure to stop the revelation from falling into the hands of Gesti.
-QUEST_LV_0100_20150323_006420	Go find Gesti. {nl}Tenet Cathedral is in Tenet Garden.
+QUEST_LV_0100_20150323_006413	$The simplest forms are the strongest ways. {nl}If you want to study the roots of everything, take the path of Elementalist.
+QUEST_LV_0100_20150323_006414	$Go straight down the Twin Bridge and you'll arrive at Srautas Gorge.{nl}Go further into the right from the Srautas Gorge and you'll be able to get to Gele Plateau.
+QUEST_LV_0100_20150323_006415	$The Paladin Master is in Gele Plateau.{nl}You can get there by riding the cable car.
+QUEST_LV_0100_20150323_006416	$This is one of the device that the first Paladin set in preparation for the invasion of the demons. {nl}Many like it are placed to cover the whole of Nefritas Cliff. {nl}
```

Comment:
"This is one of the device that the..." -> "This is one of the devices the..."

"Device" should be "devices" and remove the unnecessary "that."